database/postgres/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+VERSION="1.24.0"
+NAMESPACE="postgres"
+
+# Install cloudnative-pg operator.
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/main/releases/cnpg-${VERSION}.yaml
+kubectl rollout status deployment --namespace=cnpg-system --timeout=90s
+
+# Create the postgres database server.
+kubectl create ns ${NAMESPACE}
+kubectl -n ${NAMESPACE} apply -f .
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --for=create --timeout=60s
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --all --for=condition=ready --timeout=30s
+
+# Create the kubearchive database
+LOCAL_PORT=5433
+export PGPASSWORD=$(kubectl -n ${NAMESPACE} get secret kubearchive-superuser -o jsonpath='{.data.password}' | base64 --decode)
+kubectl -n ${NAMESPACE} port-forward service/kubearchive-rw ${LOCAL_PORT}:5432 >& /dev/null &
+pid=$!
+
+# Wait for $LOCAL_PORT to become available.
+while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
+    #echo sleeping
+    sleep 0.1
+done
+psql -h localhost -U postgres -p ${LOCAL_PORT} -f kubearchive.sql
+
+# Kill all background jobs, including the port-forward.
+trap 'kill $(jobs -p)' EXIT
```

Comment:
Can you add a comment about **why** you have to kill all the background jobs? This seems like an odd thing to do as part of an install script.

Reply:
I made the comment a little more explicit.